### PR TITLE
custom keybinder - kodi settings change via rpc - setting rewritten

### PR DESCRIPTION
**Pull request by Lantash77**
Here is some more additional changes/features.

1. Keybinder - allowing to add custom keys to control tempoup and down events.
   Select option giving 5sec window to push button for control and registering it.
   Script creating/editing custom keymap file located in userdata. Everytime updated - creating previous version backup. 
   Custom key map can be reloaded by `xbmc.executebuiltin("action(reloadkeymaps)")` but it must be used in addon - for some 
   reason not working inside service. However restart Kodi does the jobs as well. 
   `Restore defaults` button - clearing keybinds and deleting custom file.
3. Checking status enabled and setting "Sync playback to display" enabled, under Settings -> Player -> Videos.
    Changing to enable will change "Sync playback to display" to enable via rpc.
4. Additional strings also translated

Zrzut ekranu 2022-08-06 015259

### CODE REVIEW

File: service.defaultplaybackspeed.matrix/resources/language/resource.language.en_gb/strings.po [hunk 1]
```diff
@@ -61,4 +61,36 @@ msgstr ""
...
+msgid "Press the key you want to assign now"
+msgstr ""
+
+msgctxt "#32112"
+msgid "Timeout in %.0f seconds..."
+msgstr ""
+
+msgctxt "#32113"
+msgid "Custom keybind"
+msgstr "Przypisanie klawiszy"
```
Comment (znedw) [inline service.defaultplaybackspeed.matrix/resources/language/resource.language.en_gb/strings.po]: accident?

**Comment by znedw** [comment]
looks good to me, you need to merge the main branch of my repo into yours as it can't resolve some conflicts

**Comment by Lantash77** [comment]
Oh
Sorry, yes accident.
Thank you
________________________________

**Comment by Lantash77** [comment]
Ok, will try that way to solve conflicts.
To be ownest never try that before.
Merge your into main - solve conflicts - and then PR?
________________________________

**Comment by znedw** [comment]
I think the easiest way is go to the your fork and press the `Sync fork` button
image

**Comment by znedw** [comment]
No need to do another PR, this one will update

**Comment by Lantash77** [comment]
Didn't have sync button.
Seems I reversed commit too.
Will try when get to computer.

________________________________

**Pull request closed without merging**